pkg/scheduler/internal/cache/comparer/cache_comparer.go

```diff
@@ -29,31 +29,33 @@ import (
 	internalqueue "k8s.io/kubernetes/pkg/scheduler/internal/queue"
 )
 
-type cacheComparer struct {
-	nodeLister corelisters.NodeLister
-	podLister  corelisters.PodLister
-	cache      schedulerinternalcache.Cache
-	podQueue   internalqueue.SchedulingQueue
-	compareStrategy
+// CacheComparer is an implementation of the Scheduler's cache comparer.
+type CacheComparer struct {
+	NodeLister corelisters.NodeLister
+	PodLister  corelisters.PodLister
+	Cache      schedulerinternalcache.Cache
+	PodQueue   internalqueue.SchedulingQueue
+	CompareStrategy
```

Comment:
Personally, I'd like this to be the first line of struct followed with a empty line to distinguish normal field.

Reply:
Yes, I agree with you. But maybe we need a new PR for this.

Reply:
I think it's fine in this PR since this type is moving out of the package that needed to use it. And when a package has basically only one type to export, having a function called simply `New` is fine.